.github/workflows/ocwm-pre-meeting-check.yml

```
@@ -0,0 +1,83 @@
+name: OCWM Pre-Meeting Check
+
+on:
+  schedule:
+    - cron: '50 18 15-21 * 1'  # Runs at 11:50 AM PT on the 3rd Monday of the month
+
+jobs:
+  check_agenda:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v4
+
+    - name: Set up Node 20
+      uses: actions/setup-node@v4
+      with:
+        node-version: '20'
+
+    - name: Get GitHub Token
+      uses: actions/create-github-app-token@v1
+      id: get_workflow_token
+      with:
+        app-id: ${{ vars.APP_ID }}
+        private-key: ${{ secrets.PRIVATE_KEY }}
+
+    - name: Install Dependencies
+      run: npm install @octokit/core@5.1.0
+
```

```suggestion

    # Step to check if today is the third Monday
    - name: Check if today is the third Monday
      id: check-third-monday
      run: |
        day=$(date +%d)
        dow=$(date +%u)  # Day of the week (1 = Monday, ..., 7 = Sunday)
        # Check if the day is between 15th and 21st, and if it's Monday (1)
        if [ "$dow" -ne 1 ]; then
          echo "Not a Monday, exiting..."
          exit 0
        fi
        if [ "$day" -ge 15 ] && [ "$day" -le 21 ]; then
          echo "This is the third Monday of the month!"
        else
          echo "Not the third Monday, exiting..."
          exit 0
        fi
        
```

This new step will make sure the process only runs completely the third monday of the month.